backport.functions_tests

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
 
 set -eu
-SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+SCRIPT_DIR=$(readlink -f "$0")
```

Comment:
Why this change?

Reply:
> Why this change?

To fix https://www.shellcheck.net/wiki/SC2312

> Consider invoking this command separately to avoid masking its return value (or use '|| true' to ignore).shellcheck(2312)